QUDTcontributions2023.ttl

```diff
@@ -54,6 +54,102 @@ unit:KiloM2
   rdfs:label "Square Kilometer"@en-us ;
   rdfs:label "Square Kilometre"@en ;
 .
+unit:GM-PER-HectoGM
+  a qudt:Unit ;
+  dcterms:description "0,001-fold of the SI base unit kilogram divided by 0,1-fold of the SI base unit kilogram" ;
+  qudt:plainTextDescription "0,001-fold of the SI base unit kilogram divided by 0,1-fold of the SI base unit kilogram" ;
+  qudt:applicableSystem sou:CGS ;
+  qudt:applicableSystem sou:CGS-EMU ;
+  qudt:applicableSystem sou:CGS-GAUSS ;
+  qudt:applicableSystem sou:SI ;
+  qudt:conversionMultiplier 0.01 ;
+  qudt:expression "\\(g-hg^{-1}\\)"^^qudt:LatexString ;
+  qudt:hasDimensionVector qkdv:A0E0L0I0M0H0T0D0 ;  
+  qudt:qkdvNumerator qkdv:A0E0L0I0M1H0T0D0 ;
+  qudt:qkdvDenominator qkdv:A0E0L0I0M-1H0T0D0 ;
+  qudt:hasQuantityKind quantitykind:MassRatio ;
+  qudt:symbol "g/hg" ;
+  rdf:type qudt:Unit ;
+  qudt:ucumCode "g.hg-1"^^qudt:UCUMcs ;
+  qudt:ucumCode "g/hg"^^qudt:UCUMcs ;
+  rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit> ;
+  rdfs:label "Gram Per Hectogram"@en ;
+
+unit:CentiM3-PER-CentiM3
+  a qudt:Unit ;
+  dcterms:description "A common metric unit of volume. One cubic centimeter equals exactly 0.000001 cubic meters." ;
+  qudt:applicableSystem sou:CGS ;
+  qudt:applicableSystem sou:CGS-EMU ;
+  qudt:applicableSystem sou:CGS-GAUSS ;
+  qudt:applicableSystem sou:SI ;
+  qudt:conversionMultiplier 0.000001 ;
+  qudt:expression "\\(cm^{3}-cm^{-3}\\)"^^qudt:LatexString ;
+  qudt:qkdvNumerator qkdv:A0E0L3I0M0H0T0D0 ;
+  qudt:qkdvDenominator qkdv:A0E0L-3I0M0H0T0D0 ;
+  qudt:symbol "cm3/cm3" ;
+  rdf:type qudt:Unit ;
+  qudt:ucumCode "cm3.cm-3"^^qudt:UCUMcs ;
+  qudt:ucumCode "cm3/cm3"^^qudt:UCUMcs ;
+  rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit> ;
+  rdfs:label "Cubic Centimeter Per Cubic Centimeter"@en ;
+
+unit:MicroGM-PER-L
+  a qudt:Unit ;
+  dcterms:description "0,000000001-fold of the SI base unit kilogram divided by the unit litre" ;
+  qudt:applicableSystem sou:CGS ;
+  qudt:applicableSystem sou:CGS-EMU ;
+  qudt:applicableSystem sou:CGS-GAUSS ;
+  qudt:applicableSystem sou:SI ;
+  qudt:conversionMultiplier 0.000000001 ;
+  qudt:expression "\\(cm^{3}-cm^{-3}\\)"^^qudt:LatexString ;
+  qudt:qkdvNumerator qkdv:A0E0L0I0M1H0T0D0 ;
+  qudt:qkdvDenominator qkdv:A0E0L-3I0M0H0T0D0 ;

```

Comment:
This should be L3, not L-3, if you choose to make the numerator and denominator explicit. Either way, you still need qudt:hasDimensionVector qkdv:A0E0L-3I0M1H0T0D0 ;